src/Health/HealthController.ts

```diff
@@ -40,6 +45,7 @@ export const isFailureHard = (failureType: string) => failureType === 'HARD'
 
 export interface Dependencies {
   readonly healthDAO: HealthDAO
+  readonly ipfsDirectoryHasDAO: IPFSDirectoryHashDAO
```

Comment:
typo I think. HasDAO -> HashDAO? @krobi64

Reply:
🤦‍♀️

Reply:
I'll fix in the next PR./